### PR TITLE
Add test to l1-l2 messaging

### PR DESCRIPTION
**Pull request by Nathan-SL**

### CODE REVIEW

File: test/postman.test.ts [hunk 1]
```diff
@@ -166,4 +166,35 @@ describe("Postman", function () {
...
+            0
+        );
+
+        expect(l1ToL2Response).haveOwnProperty("transaction_hash");
+        await account.invoke(l2contract, "increase_balance", {
+            user,
+            amount: 10000000
+        });
+
+        await account.invoke(l2contract, "withdraw", {
```
Comment (FabijanC) [inline test/postman.test.ts:187]: What is the purpose of these two invokes?

**Review by FabijanC — CHANGES_REQUESTED** [review]

File: test/postman.test.ts [hunk 2]
```diff
@@ -166,4 +166,35 @@ describe("Postman", function () {
...
+
+        const l1ToL2Response = await starknet.devnet.sendMessageToL2(
+            l2contract.address,
+            "deposit",
+            L1_CONTRACT_ADDRESS,
+            [1, 1],
+            0
+        );
+
+        expect(l1ToL2Response).haveOwnProperty("transaction_hash");
```
[the remark this answers is not in the document]
Comment (mikiw) [reply test/postman.test.ts:181]: We could also check if the transaction was accepted on L2. In devnet there is something like this `    assert_tx_status(response.json().get("transaction_hash"), "ACCEPTED_ON_L2")`

File: test/postman.test.ts (hunk 1 above)
Comment (mikiw) [reply test/postman.test.ts:187]: These invokes are analogical to our test in devnet -> `def test_consume_message_from_l2_deploy_execute()`

**Review by mikiw — CHANGES_REQUESTED** [review]

File: test/postman.test.ts [hunk 3]
```diff
@@ -166,4 +166,35 @@ describe("Postman", function () {
...
+            amount: 10000000
+        });
+
+        await account.invoke(l2contract, "withdraw", {
+            user,
+            amount: 10,
+            L1_CONTRACT_ADDRESS: BigInt(l1l2Example.address)
+        });
+
+        const l2toL1Response = await starknet.devnet.sendMessageToL1(
```
[the remark this answers is not in the document]
Comment (mikiw) [reply test/postman.test.ts:193]: I agree it should be called `consumeMessageFromL2` since we are not sending anything we are just mocking it on L2.

**Review by FabijanC — APPROVED** [review]

**Review by mikiw — APPROVED** [review]

**Pull request merged**